src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -98,34 +60,45 @@ public static ContainedDocument TryGetContainedDocument(DocumentId id)
             ITextBuffer dataBuffer,
             IVsTextBufferCoordinator bufferCoordinator,
             Workspace workspace,
-            VisualStudioProject project,
```

Comment:
Check this with TypeScript: they call into either this or ContainedLanguage constructors.